src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -24,7 +24,6 @@ import * as User from '../../../libs/actions/User';
 import FormHelpMessage from '../../../components/FormHelpMessage';
 import MagicCodeInput from '../../../components/MagicCodeInput';
 import Terms from '../Terms';
-import DotIndicatorMessage from '../../../components/DotIndicatorMessage';
 import PressableWithFeedback from '../../../components/Pressable/PressableWithFeedback';
 import usePrevious from '../../../hooks/usePrevious';
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/21390 this implementation causes that if you go back and sign in again with the same email new magic code is sent as the timer got reset. I think its just a polish or maybe its something we are intentionally happy with but noting here https://github.com/Expensify/App/issues/21390

Reply:
This is the expected behaviour

> Press the blue Go back link before the timer ends and re-enter your email
> Confirm the timer has restarted on the magic code view and that you receive another magic code notification